_includes/workshops/icml/icml2021.html

```diff
@@ -16,12 +16,13 @@ <h2 class="blog-title">Extended Abstracts</h2>
         <div class="blog-post schedule">
         <table rules="rows" class="table" style="table-layout: fixed; width: 100%">
           <colgroup>
-            <col style="width: 5%">
+            <col style="width: 10%">
             <col style="width: 35%">
```

Comment:
@anthxnyR the title column width should be increased more so that the category tabs remain on one line and are not stretched.